songbank-backend/backend.js

```diff
@@ -134,170 +134,170 @@ const playlists = {
-        },
-        {
-            title: 'Ghost',
-            Artist: 'Justin Bieber',
-            runtime: 2.33,
-        },
-        {
-            title: 'Happier Than Ever',
-            Artist: 'Billie Eilish',
-            runtime: 2.31,
-        },
-        {
-            title: 'good 4 u',
-            Artist: 'Olivia Rodrigo',
-            runtime: 2.58,
-        },
-        {
-            title: 'Need to Know',
-            Artist: 'Doja Cat',
-            runtime: 3.31,
-        },
-        {
-            title: 'MONTERO(Call Me By Your Name',
-            Artist: 'Lil Nas X',
-            runtime: 2.18,
-        },
-        {
-            title: 'Life Goes On',
-            Artist: 'Oliver Tree',
-            runtime: 2.42,
-        },
-        {
-            title: 'Pepas',
-            Artist: 'Farruko',
-            runtime: 4.47,
-        },
-        {
-            title: "Don't Be Shy",
-            Artist: 'Tiesto',
-            featured: 'KAROL G',
-            runtime: 2.21,
-        },
-        {
-            title: 'Let Somebody Go',
-            Artist: 'Coldplay',
-            featured: 'Selena Gomez',
-            runtime: 4.02,
-        },
-        {
-            title: 'Take My Breath',
-            Artist: 'The Weeknd',
-            runtime: 3.40,
-        },
-        {
-            title: 'Kiss Me More',
-            Artist: 'Doja Cat',
-            featured: 'SZA',
-            runtime: 3.29,
-        },
-        {
-            title: 'Levitating',
-            Artist: 'Dua Lipa',
-            featured: 'DaBaby',
-            runtime: 3.23,
-        },
-        {
-            title: 'OUT OUT',
-            Artist: 'Joel Corry',
-            featured: 'Jax Jones',
-            runtime: 2.43,
-        },
-        {
-            title: 'Meet Me At Our Spot',
-            Artist: 'THE ANXIETY',
-            featured: 'Willow',
-            runtime: 2.43,
-        },
-        {
-            title: 'Livin It Up',
-            Artist: 'Young Thug',
-            featured: 'Post Malone',
-            runtime: 3.31,
-        },
-        {
-            title: 'traitor',
-            Artist: 'Olivia Rodrigo',
-            runtime: 3.49,
-        },
-        {
-            title: 'Where Are You Now',
-            Artist: 'Lost Frequencies',
-            featured: 'Calum Scott',
-            runtime: 2.28,
-        },
-    ]
+        [
+            {
+                title: 'Easy On Me',
+                Artist: 'Adele',
+                runtime: 3.45,
+            },
+            {
+                title: 'Shivers',
+                Artist: 'Ed Sheeran',
+                runtime: 3.28,
+            },
+            {
+                title: 'MONEY',
+                Artist: 'LISA',
+                runtime: 2.48,
+            },
+            {
+                title: 'STAY',
+                Artist: 'The Kid LAROI',
+                featured: 'Justin Bieber',
+                runtime: 2.22,
+            },
+            {

```

Comment:
We can remove hardcoded elements now that we have our backend working with our remote database